discord/abc.py

```diff
@@ -1493,8 +1493,8 @@ async def send(
             .. versionadded:: 1.4
 
         reference: Union[:class:`~discord.Message`, :class:`~discord.MessageReference`, :class:`~discord.PartialMessage`]
-            A reference to the :class:`~discord.Message` to which you are replying, this can be created using
-            :meth:`~discord.Message.to_reference` or passed directly as a :class:`~discord.Message`. You can control
+            A reference to the :class:`~discord.Message` you are replying to or forwarding, this can be created using
+            :meth:`~discord.Message.to_reference` or passed directly as a :class:`~discord.Message`. When replying, you can control
```

Comment:
```suggestion
            A reference to the :class:`~discord.Message` you are replying to or forwarding, this can be created using
            :meth:`~discord.Message.to_reference` or passed directly as a :class:`~discord.Message`. When replying, you can control
```
Two things here
- this is a run-on sentence. Please reword and fix.
- "you" is too casual and the sentence should be reworded in passive voice.